apps/sms/views/conversation/test/unit/conversation_test.js

```diff
@@ -5110,14 +5110,10 @@ suite('conversation.js >', function() {
   });
 
   suite('Sending Behavior (onSendClick)', function() {
-    function clickButton() {
-      clickButtonAndSelectSim(0);
-    }
-
-    function clickButtonAndSelectSim(serviceId) {
+    function clickButtonAndSelectSim(serviceId = 0) {
       ConversationView.onSendClick();
       // we get a string from the MultiSimActionButton
-      ConversationView.simSelectedCallback(undefined, '' + serviceId);
```

Comment:
@azasypkin  That's why i removed phoneNumber argument as it was not used anywhere in code.